<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shv::visu::timeline::GraphProbeWidget</class>
 <widget class="QWidget" name="shv::visu::timeline::GraphProbeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="fHeader">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbClose">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/close.svg</normaloff>:/images/close.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teCurrentTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string notr="true">yyyy-MM-dd hh:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbPrevSample">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/shv/visu/images/media-step-backward.svg</normaloff>:/shv/visu/images/media-step-backward.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbNextSample">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/shv/visu/images/media-step-forward.svg</normaloff>:/shv/visu/images/media-step-forward.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="twData">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
